src/fsharp/pars.fsy

```diff
@@ -1924,6 +1924,11 @@ atomicPatternLongIdent:
     }
   | GLOBAL DOT pathOp  { let (LongIdentWithDots(lid,dotms)) = $3 in (None,LongIdentWithDots(ident(MangledGlobalName,rhs parseState 1) :: lid, rhs parseState 2 :: dotms)) }
   | pathOp { (None,$1) }
+  | access UNDERSCORE DOT pathOp {
+    if not (parseState.LexBuffer.SupportsFeature LanguageFeature.SingleUnderscorePattern) then
```

Comment:
I'm afraid that FSharp47 has shipped, and so no additional features can be added to it:
The next release of fsharp will be fsharp5 and so this will need to be targeted at release\fsharp5.

There needs to be a new LanguageFeatureId perhaps SingleUnderscoreMethodPattern.

And for testing there should be a test case for when langversion4.6 is selected.  In that case it should produce an error
You can add those tests here fsharp/tests/fsharp/core/members/self-identifier/version46/test.fs

Reply:
Are you saying this won't be available until F# 5.0 ?

I thought it was going to be shipped in the first update for 4.7 as it's something that was missing from the original feature.

Reply:
We should just take this as a bug fix for F# 4.7, not treat it as a new language feature.